plugins/evaluation/__init__.py

```diff
@@ -988,12 +988,22 @@ def resolve_input(self, ctx):
 
         get_eval_key(ctx, inputs)
 
+        inputs.bool(
+            "select_fields",
+            default=False,
+            label="Select fields",
+            description="Exclude fields involved in other evaluations",
```

Comment:
suggestion: I like adding a `?` at the end of the description

Reply:
done